deploy/terraform/providers/aws/variables.tf

```diff
@@ -46,7 +46,7 @@ variable "min_capacity" {
 }
 
 variable "instance_type" {
-  default = "t2.medium"
+  default = "t2.xlarge"
```

Comment:
why the need to go bigger bro?

Reply:
Primarily for the ETH node. Had issues with a medium where pods were getting evicted.

Reply:
ok got it.